org-formation/050-costs/_tasks.yaml

```diff
@@ -68,5 +68,6 @@ CostNotificationMicroservice:
     Region: !Ref primaryRegion
   Parameters:
     AdminEmail: "it@sagebase.org"
+    SkipRecipients: "bgrande@synapse.org,bruno.grande@sagebase.org,kelsey.montgomery@sagebase.org,larsson.omberg@sagebase.org,larssono@synapse.org,tess.thyer@sagebase.org"
```

Comment:
Yes, but we are also responsible for not sending emails that we know will bounce, so we need to do both. The cost category will be updated within a day of the tags updating, so even if the tags are updated immediately when a tagged owner leaves Sage, we can expect a report to be generated for that ex-employee for the portion of the month they were a resource owner. 

And I agree that this is difficult to maintain, but we should also try to keep it as short as possible by updating tags early so that the corresponding recipients can be removed after their last monthly email.